types/index.d.ts

```diff
@@ -0,0 +1,33 @@
+declare module WebdriverIO {
```

Comment:
I don't know if it's possible, but can you use the existing types for the options?

```js
import { SaveFullPageMethodOptions } from 'webdriver-image-comparison/build/commands/fullPage.interfaces';
import { SaveScreenMethodOptions } from 'webdriver-image-comparison/build/commands/screen.interfaces';
import { SaveElementMethodOptions } from 'webdriver-image-comparison/build/commands/element.interfaces';
```

Reply:
Let me give it another go. :) I was having some trouble using them.